tests/unit/test_repos_repo.py

```diff
@@ -58,6 +58,64 @@ def test_asset_requires_a_positive_id(self):
 
         assert self.session.get.called is False
 
+    def test_create_fork(self):
+        """Verify the request to fork a repository."""
+        self.instance.create_fork()
+        self.post_called_with(
+            url_for('forks')
+        )
+
+    def test_create_hook(self):
+        """Verify the request to create a hook."""
+        data = {
+            'name': 'web',
+            'config': {
+                'url': 'http://example.com/webhook',
+                'content_type': 'json'
+            }
+        }
+
+        self.instance.create_hook(**data)
+        self.post_called_with(
+            url_for('hooks'),
+            data={
+                'name': 'web',
+                'config': {
+                    'url': 'http://example.com/webhook',
+                    'content_type': 'json'
+                },
+                'events': ['push'],
+                'active': True
+            }
+        )
+
+    def test_create_issue(self):
+        """Verify the request to create an issue."""
+        data = {
+            'title': 'Unit Issue',
+            'body': 'Fake body',
+            'assignee': 'sigmavirus24',
+            'milestone': 1,
+            'labels': ['bug', 'enhancement']
+        }
+        self.instance.create_issue(**data)
```

Comment:
We probably could use some test cases here to ensure that our (probably lightweight) validation doesn't allow through certain values.